test/fixtures/flow/anonymous-function-no-parens-types/bad_01/actual.js

```diff
@@ -0,0 +1 @@
+var f = (x): number => 123 => 123;
```

Comment:
Yeah, it's not easy to do. The problem is that the first `=>` is parsed as arrow function's `=>`. So we'd have to decide which parsing errors after the `=>` would need this explanation and which are just normal arrow function parse errors.